docs/guides/js-waku/relay-send-receive.md

```diff
@@ -0,0 +1,100 @@
+---
+title: Send and Receive Messages Using Relay
+---
+
+This guide provides detailed steps to set up a Waku node for sending and receiving messages using the [Relay protocol](/overview/concepts/protocols#relay).
+
+## Create a Waku Node
+
+Set up a Waku node by creating a relay node, connecting to network peers, choosing a [content topic](/overview/concepts/content-topics), and creating an `encoder` and `decoder` for [message encryption](https://rfc.vac.dev/spec/26/):
+
+```js
+import {
+	createRelayNode,
+	waitForRemotePeer,
+	createEncoder,
+	createDecoder
+} from "@waku/sdk";
+
+// Create and start a relay node
+const node = await createRelayNode({
+	defaultBootstrap: true, // bootstraps using pre-defined nodes
+	emitSelf: true, // emits sent message events to itself
+});
+await node.start();
+
+// Wait for a successful peer connection
+await waitForRemotePeer(node);
+
+// Choose a content topic
+const contentTopic = "/relay-guide/1/message/proto";
+
+// Create a message encoder and decoder
+const encoder = createEncoder({ contentTopic });
+const decoder = createDecoder(contentTopic);
```

Comment:
I would add a tip here:

In this example, all users send and receive messages on the same content topic.
A real application may have some user broadcast messages on a content topic while others listent to them, or only have 1:1 exchanges, or both. Waku supports all these uses cases.